tests/kafkatest/services/util.py

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Doesn't have to happen now, but this is probably useful and general enough to graduate to ducktape